pkg/manifests/manifests.go

```diff
@@ -1657,6 +1657,9 @@ func (f *Factory) setupQueryLogFile(p *monv1.Prometheus, queryLogFile string) er
 		return nil
 	}
 	dirPath := filepath.Dir(queryLogFile)
+	if dirPath != "." && strings.HasPrefix(dirPath, ".") {
```

Comment:
Can we use [filepath#Abs](https://pkg.go.dev/path/filepath#Abs)?
```suggestion
	if filepath.Abs(queryLogFile) != queryLogFile {
```

Reply:
Ended up using a combination of this with `dirPath != "."` since we have to account for the case where the `queryLogFile` its just a filename